pkg/blockatlas/client.go

```diff
@@ -5,14 +5,15 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"go.elastic.co/apm/module/apmhttp"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"net/url"
 	"strings"
 	"time"
 
+	"go.elastic.co/apm/module/apmhttp"
```

Comment:
do not inject any dependency here

Reply:
Sorry, this was like that.  I just added an extra error check to this generic file, not changing it's structure.